torch/csrc/jit/python/init.cpp

```diff
@@ -1498,9 +1498,34 @@ void initJITBindings(PyObject* module) {
             const AliasInfo* aliasInfo = self.alias_info();
             return aliasInfo && aliasInfo->isWrite();
           })
+      .def_property_readonly(
```

Comment:
nit: since we have so many properties that we want to get from AliasInfo, maybe we should bind the AliasInfo type and then add these properties onto that, just so that the python code doesn't look too far off from the c++ code (you can do this as a separate PR if you prefer)

Reply:
Sure! I'll do that in a separate PR just so this one doesn't get too cluttered.